configure.ac

```diff
@@ -8,7 +8,7 @@ dnl ============================================================================
 dnl Custom macro definitions.
 
 dnl JE_CONCAT_VVV(r, a, b)
-dnl 
```

Comment:
```dnl ``` is an m4 comment delimiter.  ```dnla``` is not a comment delimiter, and IIRC best practice is to always follow ```dnl``` by a space.